main.py

```diff
@@ -284,8 +287,33 @@ def search_key():
             windowSurface.blit(KEYPIC,box)
             pygame.display.flip()
             pygame.time.delay(1000)
+	    show_end_screen()
 
         pygame.display.flip()
 
+def show_end_screen():
+    BG=pygame.image.load('Images/jail background.jpg')
+    windowSurface.blit(pygame.transform.scale(BG,(size)),(0,0))
+    backtext = pygame.font.SysFont("liberationserif",40)
+    textSurf, textRect = text_objects("Dare to play again!!", backtext)
+    textRect.center = ( (width/2), (height/2) - 200)
+    windowSurface.blit(textSurf, textRect)
+    textSurf, textRect = text_objects("Press space to play again", backtext)

```

Comment:
Maybe make a 'Play again' button and a 'Go back' button like the start screen has for consistency.